core/src/test/java/io/confluent/kafka/schemaregistry/rest/RestApiTest.java

```diff
@@ -590,6 +591,21 @@ public void testGetSubjectsAssociatedWithNotFoundSchemaId() throws Exception {
     }
   }
 
+  @Test
+  public void testGetVersionsAssociatedWithSchemaId() throws Exception {
+    String subject1 = "testTopic1";
+    String subject2 = "testTopic2";
+
+    String schema = TestUtils.getRandomCanonicalAvroString(1).get(0);
+    TestUtils.registerAndVerifySchema(restApp.restClient, schema, 1, subject1);
+    TestUtils.registerAndVerifySchema(restApp.restClient, schema, 1, subject2);
+
+    List<SubjectVersion> associatedSubjects = restApp.restClient.getAllVersionsById(1);
+    assertEquals(associatedSubjects.size(), 2);
+    assertEquals(Arrays.asList(new SubjectVersion(subject1, 1), new SubjectVersion(subject2, 1)),
+        associatedSubjects);
```

Comment:
Do we know for sure the list has the same order as returned by the underlying map ?

Reply:
Good point, I'll fix it